hadoop-ozone/integration-test-s3/src/test/java/org/apache/hadoop/ozone/s3/awssdk/v1/TestS3SDKV1WithRatisStreaming.java

```diff
@@ -36,6 +38,7 @@ public static void init() throws Exception {
         false);
     conf.setBoolean(OzoneConfigKeys.OZONE_NETWORK_TOPOLOGY_AWARE_READ_KEY,
         true);
+    conf.set(HDDS_DATANODE_VOLUME_MIN_FREE_SPACE, "5GB");
```

Comment:
Please don't add this in each integration test.  Set it in `hadoop-ozone/integration-test-s3/src/test/resources/ozone-site.xml`.

Similarly, add it in `hadoop-ozone/integration-test-recon/src/test/resources/ozone-site.xml` (merge from `master` first).